README.adoc

```diff
@@ -81,7 +81,12 @@ If you are running tutorial-web-app against your cluster, then you will need to
 $ oc login <my-cluster>
 $ oc patch oauthclient tutorial-web-app -p '{"redirectURIs":["http://localhost:3006"]}'
 ----
-
+The operator will reset this redirect url so you should stop the operator while testing. 
```

Comment:
Best to mention that its the integreatly operator that will do this and not the webapp operator.

Reply:
agreed. Will update to include this.